src/sparseml/export/export_data.py

```diff
@@ -63,30 +63,44 @@ def create_data_samples(
     inputs, outputs, labels = [], [], []
     if model is None:
         _LOGGER.warning("The model is None. The list of outputs will be empty")
-    for batch_num, (inputs_, labels_) in tqdm(enumerate(data_loader)):
+
+    for batch_num, data in tqdm(enumerate(data_loader)):
         if batch_num == num_samples:
             break
-        if model:
-            outputs_ = model(inputs_)
+
+        if isinstance(data, dict):
+            # assume transformers inference
```

Comment:
can we move transformers specific code out of here somehow?

Reply:
Yeah